proposals/semi-auto-properties.md

```diff
@@ -156,6 +156,50 @@ public class Point
     - https://github.com/dotnet/csharplang/issues/5563
     - https://github.com/dotnet/csharplang/pull/5573#issuecomment-1002110830
 
+### Interaction with partial properties
+
+#### Initializers
+
+When a partial property uses `field`, which parts should be allowed to have an initializer?
+
+```cs
+partial class C
+{
+    public partial int Prop { get; set; } = 1;
+    public partial int Prop { get => field; set => field = value; } = 2;
+}
+```
+
+- It seems clear that an error should occur when both parts have an initializer.
+- We can think of use cases where either the definition or implementation part might want to set the initial value of the `field`.
+- It seems like if we permit the initializer on the definition part, it is effectively forcing the implementer to use `field` in order for the program to be valid. Is that fine?
+- We think it will be common for generators to use `field` whenever a backing field of the same type is needed in the implementation. This is in part because generators often want to enable their users to use `[field: ...]` targeted attributes on the property definition part. Using the `field` keyword saves the generator implementer the trouble of "forwarding" such attributes to some generated field and suppressing the warnings on the property. Those same generators are likely to also want to allow the user to specify an initial value for the field.
+
+**Recommendation**: Permit an initializer on either part of a partial property when the implementation part uses `field`. Report an error if both parts have an initializer.
+
+#### Auto-accessors
+
+As originally designed, partial property implementation must have bodies for all the accessors. However, recent iterations of the `field` keyword feature have included the notion of "auto-accessors". Should partial property implementations be able to use such accessors? If they are used exclusively, it will be indistinguishable from a defining declaration.
```

Comment:
> However, recent iterations of the `field` keyword feature have included the notion of "auto-accessors".

Was there an explicit language design decision to allow a property to contain an "auto-accessor" *and* an explicitly implemented accessor? I couldn't find a reference in the LDM notes.

Reply:
#8069 says

> Properties may now mix and match auto accessors with full accessors.